lib/hiera/config.rb

```diff
@@ -10,8 +10,8 @@ class << self
     # @return [Hash] representing the configuration.  e.g.
     #   {:backends => "yaml", :hierarchy => "common"}
```

Comment:
could you update this comment, and a comment above this about what the default behavior is supposed to be?